kingpin/actors/rightscale/test/test_base.py

```diff
@@ -130,22 +130,16 @@ def test_generate_rightscale_params(self):
 
     def test_generate_rightscale_params_with_array(self):
         self.maxDiff = None
-        params = {'name': 'unittest-name',
-                  'status': 'enabled',
-                  'elasticity_params': {
-                      'schedule': [
-                          {'day': 'Sunday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'},
-                          {'day': 'Monday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'},
-                          {'day': 'Tuesday', 'max_count': 2,
-                           'min_count': 1, 'time': '07:00'}
-                      ]
-                  }}
-        expected_params = [
-            ('server_array[status]', 'enabled'),
-            ('server_array[name]', 'unittest-name'),
+        schedule = [
+            {'day': 'Sunday', 'max_count': 2, 'min_count': 1, 'time': '07:00'},
+            {'day': 'Monday', 'max_count': 2, 'min_count': 1, 'time': '07:00'},
+            {'day': 'Tuesday', 'max_count': 2, 'min_count': 1, 'time': '07:00'}
```

Comment:
Doesn't this test assume that assigning to a dict changes the order in which items are returned? Wouldn't it be a stronger test if each dict were an OrderedDict with its items in a different order?